frontend/src/views/Pages/UserProfile/UserCard_CoinAnimation.spec.js

```diff
@@ -37,6 +37,7 @@ describe('UserCard_CoinAnimation', () => {
 
   describe('mount', () => {
     beforeEach(() => {
+      jest.clearAllMocks()
```

Comment:
As long as you not clear the mocks, the tests for store commit and toaster are always true, because there is an call before with the other values @ulfgebhardt